Code/Editor/EditorViewportSettings.cpp

```diff
@@ -588,4 +606,25 @@ namespace SandboxEditor
     {
         AzToolsFramework::SetRegistry(CameraFarPlaneDistanceSetting, aznumeric_cast<double>(distance));
     }
+
+    float CameraDefaultFovRadians()
+    {
+        return aznumeric_caster(AzToolsFramework::GetRegistry(CameraFovSetting, aznumeric_cast<double>(AZ::DegToRad(60.0))));
+    }
+
+    void SetCameraDefaultFovRadians(float fovRadians)
+    {
+        AzToolsFramework::SetRegistry(CameraFovSetting, aznumeric_cast<double>(fovRadians));
+    }
+
+    float CameraDefaultFovDegrees()
+    {
+        return AZ::RadToDeg(CameraDefaultFovRadians());
+    }
+
+    void SetCameraDefaultFovDegrees(float fovDegrees)
+    {
+        SetCameraDefaultFovRadians(AZ::DegToRad(fovDegrees));
+    }
```

Comment:
update: This is great and the right idea, but we should swap the logic so we serialize the value as degrees not radians. This will make viewing the data in the setreg file much clearer.

```
float CameraDefaultFovRadians()
{
    return AZ::DegToRad(CameraDefaultFovDegrees());
}

void SetCameraDefaultFovRadians(float fovRadians)
{
    SetCameraDefaultFovDegrees(AZ::RadToDeg(fovRadians));
}

float CameraDefaultFovDegrees()
{
    return aznumeric_caster(AzToolsFramework::GetRegistry(CameraFovDegreesSetting, 60.0));
}

void SetCameraDefaultFovDegrees(float fovDegrees)
{
    AzToolsFramework::SetRegistry(CameraFovDegreesSetting, aznumeric_cast<double>(fovDegrees));
}
```